views/js/controller/resultTable.js

```diff
@@ -215,8 +211,8 @@ define([
                 var url = $elt.data('url');
                 e.preventDefault();
                 buildGrid(url, action, function () {
-                    _.forEach(groups[group], function ($btn) {
-                        $btn.toggleClass('hidden');
+                    groups[group].forEach(function ($btn) {
+                        $btn.classList.toggle('hidden');
```

Comment:
Is this ok? `toggleClass()` is available on a jQuery element, but `classList` is a DOMElement property.
I'm assuming that `$btn` is a jQuery element because that's the convention used in this codebase.